src/views/EditUser.js

```diff
@@ -132,6 +186,73 @@ export const EditUser = (props) => {
         }
     }
 
+    const resetPwd = () => {
+        if (password.newPassword === '' && password.confirmPassword === '') {
+            // throw error if the new password is empty
+            setPassword({ ...password, status: 'Password can not be empty.' })
```

Comment:
same as other comment - let's not catch this on the clientside... this should be a policy decision.

Reply:
roger roger